### PR TITLE
Add all files

### PR DESCRIPTION
**Pull request by aster-alemu**
In this PR, I have been doin the following activities.

1. Setup the project
2. Re-create the Awesome books app by using ES6 syntax, dividing the code into modules
3. test for linter errors and fix them

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @aster-alemu , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that hasn't [linters workflow](https://github.com/microverseinc/linters-config) set up on it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  const book = new Store(title.value, author.value);
+  displayBook();
+  Stringifier();
+  title.value = '';
+  author.value = '';
+});
+
+// Add navigation
+const time = document.querySelector('#time');
+time.innerText = new Date();
```
Comment (Nyame-Wolf) [inline index.js]: Kindly note that it's a requirement to use luxon for date. as per the lesson[Awesome books: with ES6](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md)
Screenshot 2022-11-01 at 14 35 03
Please use [Luxon](https://moment.github.io/luxon/#/install?id=es6) and  import it as a module.

File: modules/stringfier.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Store from './store.js';
+
+export default function Stringifier() {
```
Comment (Nyame-Wolf) [inline modules/stringfier.js]: Kindly use es6 arrow function syntax. This is good practice and is also a requirement for this milestone
Screenshot 2022-11-01 at 14 35 03

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @aster-alemu  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great design. your project looks very nice.
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @aster-alemu,

Good job on making the changes highlighted by the previous reviewer!

Your project is complete!  

## Highlights: 
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Used ES Modules correctly as required 👍 
✅ Your app works fine as expected 👍 

## Recommendation:

- [ ] How about you add the width of the input fields a little more? They are so short in width for a book with a longer title.  For a better user experience, what do you think?

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**